src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -49,6 +52,55 @@ public abstract class AzureServiceBusMessagePump<TMessage> : MessagePump<TMessag
         /// </summary>
         protected string Namespace { get; private set; }
 
+        /// <summary>
+        /// Gets the unique identifier for this background job to distinguish this job instance in a multi-instance deployment.
+        /// </summary>
+        public string JobId { get; }
+
+        /// <summary>
+        /// Gets the name of the topic subscription; combined from the <see cref="AzureServiceBusMessagePumpSettings.SubscriptionPrefix"/> and the <see cref="JobId"/>.
+        /// </summary>
+        protected string SubscriptionName { get; }
+
+        /// <summary>
+        /// Triggered when the application host is ready to start the service.
+        /// </summary>
+        /// <param name="cancellationToken">Indicates that the start process has been aborted.</param>
+        public override async Task StartAsync(CancellationToken cancellationToken)
+        {
+            if ((Settings.Options.TopicSubscription & TopicSubscription.CreateOnStart) == TopicSubscription.CreateOnStart)
+            {
+                await CreateTopicSubscriptionAsync(cancellationToken);
+            }
+
+            await base.StartAsync(cancellationToken);
+        }
+
+        private async Task CreateTopicSubscriptionAsync(CancellationToken cancellationToken)
```

Comment:
Thinking out loud - We could move this to a `AzureServiceBusTopicManager` or so, but not a must.